blocks/pull-quote/pull-quote.css

```diff
@@ -0,0 +1,20 @@
+main .pull-quote {
+    border-left: 3px solid #367ea8;
```

Comment:
I think this should be set as variable from styles.css, Not sure what would be the name. I'd like to know if we are going to have a style guide someday to refer.

Reply:
i think that's a great idea... in absence of any design guidance i would propose to use:

https://github.com/adobe/business-website/blob/main/styles/styles.css#L17

IIRC the blog design correctly, this color was only used in the pull quotes though, but as it is used at least for both the text and the border i think it might even make sense to define a var locally to the block.

Reply:
Thanks! I will look forward to have some utility classes for common styles.